### PR TITLE
21.3 French translation and other changes

### PR DESCRIPTION
**Pull request by FJBDev**
- 21.3 French translation
- Unit tests update (removal of the srtmSerie per the new [SRTM policy](https://github.com/wolfgang-ch/mytourbook/issues/309))
- Adding the ability to cancel the download and upload for the cloud vendors
- Misc code improvements:
  - [Array designators "[]" should be located after the type in method signatures](https://rules.sonarsource.com/java/tag/convention/RSPEC-1195)
  - [Removing unused parameters](https://rules.sonarsource.com/java/tag/unused/RSPEC-1172)
  - [Making anonymous inner class a lambda](https://rules.sonarsource.com/java/RSPEC-1604)

Unit tests 100% successful

image

### CODE REVIEW

File: bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java [hunk 1]
```diff
@@ -3473,16 +3446,14 @@ private void drawAsync_530_BarGraph(final GC gcGraph, final GraphDrawingData dra
              * make sure the bars do not overlap
              */
             if (serieLayout != ChartDataYSerie.BAR_LAYOUT_SINGLE_SERIE) {
-               if (devXPosNextBar > 0) {
-                  if (devXPos < devXPosNextBar) {
+               if (devXPosNextBar > 0 && devXPos < devXPosNextBar) {
 
-                     // bars do overlap
+                  // bars do overlap
```
Comment (wolfgang-ch) [inline bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java:3452]: The comment `// bars do overlap` is only for the inner `if` of the old code and not for the joined `if`

Comment (FJBDev) [reply bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java:3452]: Would that be more accurate like this ?

image

Comment (wolfgang-ch) [reply bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java:3452]: > Would that be more accurate like this ?

Yes

Comment (FJBDev) [reply bundles/net.tourbook.chart/src/net/tourbook/chart/ChartComponentGraph.java:3452]: Ok, I fixed it in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/313)

**Pull request merged**